### PR TITLE
CSH-3793: Support of interactive component

### PR DESCRIPTION
**Pull request by vitaly-retel**

### CODE REVIEW

File: lib/components-schema-v1_0_x.ts [hunk 1]
```diff
@@ -293,6 +293,28 @@ export const componentsDefinitionSchema_v1_0_x = {
...
+                                properties: {
+                                    type: {
+                                        enum: ['interactive'],
+                                        description: 'Configuration of interactive directive',
+                                    },
+                                    defaultConfig: {
+                                        type: 'object',
+                                        description: 'Default configuration of interactive directive',
+                                    },
+                                    editLink: {
```
Comment (wwsno) [inline lib/components-schema-v1_0_x.ts]: Looks like it's possible to automatically validate urls:
https://spacetelescope.github.io/understanding-json-schema/reference/string.html#built-in-formats

Comment (vitaly-retel) [reply lib/components-schema-v1_0_x.ts]: yeah

Comment (vitaly-retel) [reply lib/components-schema-v1_0_x.ts]: not sure how strict it should be, I think general 'uri' should be enough

File: lib/components-schema-v1_0_x.ts [hunk 2]
```diff
@@ -305,6 +329,7 @@ export const componentsDefinitionSchema_v1_0_x = {
                             'doc-html',
                             'doc-slideshow',
                             'doc-media',
+                            'doc-interactive',
                         ],
                     },
                     dataProperty: { type: 'string' },
```
Comment (vitaly-retel) [inline lib/components-schema-v1_0_x.ts]: guess this one can be removed too?

Comment (wwsno) [reply lib/components-schema-v1_0_x.ts]: Sure.

File: lib/components-schema-v1_0_x.ts (hunk 1 above)
Comment (wwsno) [reply lib/components-schema-v1_0_x.ts]: Think so too.

**Pull request merged**